v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/stack-traces/stack-trace-solidity-errors.ts

```diff
@@ -68,7 +68,7 @@ export function createSolidityErrorWithStackTrace(
   }
 }
 
-function encodeStackTraceEntry(
+export function encodeStackTraceEntry(
```

Comment:
I export this and `getMessageFromLastStackTraceEntry` for use in `solidity-test` built-in plugin so that stack trace entry parsing doesn't have to be duplicated.